### ISSUE TITLE
Remove / Edit Item in minicart does not work if span css is changed

### ISSUE THREAD

**Issue by Zaylril** [magento/magento2#1694]
Hey, 

On our design we removed the span styles from the minicart "remove" and "edit" links which add the icon. This means that, because the span now has no styles (just shows the text) when you click the link the code in the JS to remove the item now finds the span not the a tag which contains the item id to remove and be sent with the post.

This took me a while to debug and I'm sure others may fall into the same trap. Perhaps something like this would be better.

```javascript
        _removeItem: function(elem) {
            if(elem.prop('tagName') !== 'A') {
                elem = elem.parent('a');
            }
            var itemId = elem.data('cart-item');
            this._ajax(this.options.url.remove, {
                item_id: itemId
            }, elem, this._removeItemAfter);
        },
``` 

Thanks

**Comment by okorshenko** [comment]
@Zaylril , thank you for reporting. Internal ticket MAGETWO-43229 is created to track this issue

**Comment by ilagno89** [comment]
@Zaylril, internal ticket is fixed and already available in develop branch. Please, check the fix, if something doesn't work for you, feel free to reopen this issue.

**Comment by magento-engcom-team** [comment]
Hi @Zaylril. Thank you for your report. 
The issue has been fixed in magento/magento2#29429 by @Nazar65 in 2.4-develop branch
Related commit(s):
 - [4ebe9d6397312bda6da09654c5fac51391844757](https://github.com/magento/magento2/commit/4ebe9d6397312bda6da09654c5fac51391844757)

The fix will be available with the upcoming 2.4.1 release.
